api/src/main/java/com/codeforcommunity/api/IPostsProcessor.java

```diff
@@ -0,0 +1,23 @@
+package com.codeforcommunity.api;
+
+import com.codeforcommunity.dto.Post;
+import com.codeforcommunity.dto.PostsResponse;
+
+/** Represents an object which can Process blog post requests. */
+public interface IPostsProcessor {
+
+  /**
+   * Returns all posts for the front page.
+   *
+   * @return A PostsResponse object containing a list of {@link Post}.
+   */
+  PostsResponse getPosts();
```

Comment:
When returning a list of posts, should we have a lightweight version of a post which doesn't include the body?